### PR TITLE
fix(verifier): Allow to use progress formatter during verification

### LINKED ISSUE #188: Expose formatting options for pact verification
## Software versions

-   **OS**: all 
-   **Library version**: 9.0.5
-   **Node Version**: all

## Description

Currently, it's not possible to use alternative formatting for provider verification. The underlying binary, [pact-provider-verifier](https://github.com/pact-foundation/pact-provider-verifier) allows alternative output from the verification process with the `format` option:

```
--format=FORMATTER     # RSpec formatter. Defaults to custom Pact formatter. Other options are json and RspecJunitFormatter (which outputs xml).
```

This option isn't exposed in pact-node, and should be. Relevant changes would be [here in pact-node](https://github.com/pact-foundation/pact-node/blob/master/src/verifier.ts) and [here in pact-js](https://github.com/pact-foundation/pact-js/blob/master/src/dsl/verifier.ts)

### PR DESCRIPTION
**Pull request by vgrigoruk**
Fixes #188

### CODE REVIEW

**Comment by mefellows** [comment]
Thanks - looks like the build failed at https://travis-ci.org/pact-foundation/pact-node/jobs/595583946#L121. Another reference to the rspec formatter in tests somewhere.

I'm ok with the change. It is also a breaking change, so would you mind please adding the text "BREAKING CHANGE" to the bottom of your commit message so that when we run our deployment process it automatically bumps the major version?

**Comment by TimothyJones** [comment]
I pushed a commit fixing the build failure - unless I'm missing something, I don't think this is a breaking change, @mefellows?

File: src/verifier.ts [hunk 1]
```diff
@@ -216,6 +217,6 @@ export interface VerifierOptions {
   timeout?: number;
   tags?: string[];
   monkeypatch?: string;
-  format?: 'json' | 'RspecJunitFormatter';
```
Comment (mefellows) [inline src/verifier.ts]: @TimothyJones may bad. I thought this was breaking as it was replaced by `xml`. It looks OK now, so maybe I'm going crazy.

**Comment by mefellows** [comment]
Merging - no breaking change.

**Pull request merged**